### ISSUE TITLE
[kubernetes] Adding a remote node failed

### ISSUE THREAD

**Issue by betefaber** [dojot/dojot#1548]
The remote node is not added in version **v0.5.0-alpha.2**.

```
curl -X POST 10.50.11.196:30001/flows/v1/node -H "Authorization: Bearer ${JWT}" -H "content-type: application/json" -d '{"image": "dojot/kelvin-example:3.0.0-alpha2","id":"kelvin"}' 
{"message":"Failed to add node: Failed to create docker container."}
```
flowbroker:
```
<17:09:01 08/07/2020> -- |kb8sRemoveNode -- DEBUG: Using kubernetes driver.
<17:09:01 08/07/2020> -- |kb8sRemoveNode -- DEBUG: Selected engine: kubernetes
<17:09:01 08/07/2020> -- |kb8sRemoveNode -- DEBUG: Using kubernetes API server @ https://10.96.0.1:443
<17:09:01 08/07/2020> -- |kb8sRemoveNode -- DEBUG: Testing access...
<17:09:01 08/07/2020> -- |kb8sRemoveNode -- DEBUG: Retrieving current deployment...
<17:09:01 08/07/2020> -- |kb8sRemoveNode -- DEBUG: Sending request to server...
<17:09:01 08/07/2020> -- |kb8sRemoveNode -- DEBUG: ... request was sent to the server.
<17:09:01 08/07/2020> -- |kb8sRemoveNode -- DEBUG: Retrieving current deployment...
<17:09:01 08/07/2020> -- |kb8sRemoveNode -- DEBUG: Sending request to server...
<17:09:01 08/07/2020> -- |kb8sRemoveNode -- DEBUG: ... request was sent to the server.
<17:09:01 08/07/2020> -- |kb8sRemoveNode -- DEBUG: Error: { Error: the server could not find the requested resource
    at /opt/flowbroker/orchestrator/node_modules/kubernetes-client/lib/base.js:11:21
    at Request.request [as _callback] (/opt/flowbroker/orchestrator/node_modules/kubernetes-client/lib/request.js:107:14)
    at Request.self.callback (/opt/flowbroker/orchestrator/node_modules/request/request.js:185:22)
    at emitTwo (events.js:126:13)
    at Request.emit (events.js:214:7)
    at Request.<anonymous> (/opt/flowbroker/orchestrator/node_modules/request/request.js:1161:10)
    at emitOne (events.js:116:13)
    at Request.emit (events.js:211:7)
    at IncomingMessage.<anonymous> (/opt/flowbroker/orchestrator/node_modules/request/request.js:1083:12)
    at Object.onceWrapper (events.js:313:30)
    at emitNone (events.js:111:20)
    at IncomingMessage.emit (events.js:208:7)
    at endReadableNT (_stream_readable.js:1064:12)
    at _combinedTickCallback (internal/process/next_tick.js:139:11)
    at process._tickCallback (internal/process/next_tick.js:181:9) code: 404 }
<17:09:01 08/07/2020> -- |kb8sRemoveNode -- DEBUG: Could not retrieve current deployments while creating new one.
<17:09:01 08/07/2020> -- |kb8sRemoveNode -- ERROR: Could not retrieve current deployments while creating new one. Error is Error while retrieving deployments: Error: the server could not find the requested resource
<17:09:01 08/07/2020> -- |nodeMngr -- ERROR: Failed to create docker container for
          remote node admin/kelvin ("Could not retrieve current deployments. Error is Error while retrieving deployments: Error: the server could not find the requested resource").
<17:09:01 08/07/2020> -- |kb8sRemoveNode -- DEBUG: Error: { Error: the server could not find the requested resource
    at /opt/flowbroker/orchestrator/node_modules/kubernetes-client/lib/base.js:11:21
    at Request.request [as _callback] (/opt/flowbroker/orchestrator/node_modules/kubernetes-client/lib/request.js:107:14)
    at Request.self.callback (/opt/flowbroker/orchestrator/node_modules/request/request.js:185:22)
    at emitTwo (events.js:126:13)
    at Request.emit (events.js:214:7)
    at Request.<anonymous> (/opt/flowbroker/orchestrator/node_modules/request/request.js:1161:10)
    at emitOne (events.js:116:13)
    at Request.emit (events.js:211:7)
    at IncomingMessage.<anonymous> (/opt/flowbroker/orchestrator/node_modules/request/request.js:1083:12)
    at Object.onceWrapper (events.js:313:30)
    at emitNone (events.js:111:20)
    at IncomingMessage.emit (events.js:208:7)
    at endReadableNT (_stream_readable.js:1064:12)
    at _combinedTickCallback (internal/process/next_tick.js:139:11)
    at process._tickCallback (internal/process/next_tick.js:181:9) code: 404 }
<17:09:01 08/07/2020> -- |kb8sRemoveNode -- DEBUG: ... server access is not OK.
<17:09:01 08/07/2020> -- |kb8sRemoveNode -- ERROR: Could not access Kubernetes API server: Error while retrieving deployments: Error: the server could not find the requested resource
```

![image](https://user-images.githubusercontent.com/37310063/86951191-8eafe980-c127-11ea-9b39-2c94f436e82e.png)

![image](https://user-images.githubusercontent.com/37310063/86951104-7344de80-c127-11ea-85cf-0caf8d9c4e59.png)

----
**NOTE: The remote node is added in version v0.5.0-alpha.1**
```
curl -X POST 10.50.11.97:30001/flows/v1/node -H "Authorization: Bearer ${JWT}" -H "content-type: application/json" -d '{"image": "dojot/kelvin-example:3.0.0-alpha2","id":"kelvin"}' 
{"message":"ok"}
```

```
<15:11:15 08/07/2020> -- |kb8sRemoveNode -- DEBUG: Building deployment creation request...
<15:11:15 08/07/2020> -- |kb8sRemoveNode -- DEBUG: Adding container dojot/kelvin-example:3.0.0-alpha2 to the set...
<15:11:15 08/07/2020> -- |kb8sRemoveNode -- DEBUG: ... container adminkelvin was added to the set.
<15:11:15 08/07/2020> -- |kb8sRemoveNode -- DEBUG: ... deployment creation request was built.
<15:11:15 08/07/2020> -- |kb8sRemoveNode -- DEBUG: Sending request to server...
<15:11:15 08/07/2020> -- |kb8sRemoveNode -- DEBUG: ... request was sent to the server.
<15:11:15 08/07/2020> -- |kb8sRemoveNode -- DEBUG: ... server access is OK.
<15:11:15 08/07/2020> -- |kb8sRemoveNode -- DEBUG: Creating service for this deployment...
<15:11:15 08/07/2020> -- |kb8sRemoveNode -- DEBUG: Service to be created: { apiVersion: 'v1',
  kind: 'Service',
  metadata:
   { name: 'flownode-adminkelvin',
     labels: { app: 'flowbroker', name: '' } },
  spec:
   { selector: { name: 'flownode-adminkelvin' },
     ports: [ { protocol: 'TCP', port: 5555, targetPort: 5555 } ] } }
<15:11:15 08/07/2020> -- |dispatcher -- INFO: Dispatcher connected to flownode-adminkelvin on port 5555
<15:11:22 08/07/2020> -- |dispatcher -- DEBUG: Received reply: {"payload":{"payload":{"id":"sample/kelvin","name":"kelvin","module":"kelvin","version":"0.0.0"}},"requestId":"01d7325a-f19c-4f76-b429-a8a664ce4ec5"}
<15:11:22 08/07/2020> -- |nodeMngr -- DEBUG: Succeeded to add remote node with id kelvin
```
![image](https://user-images.githubusercontent.com/37310063/86950885-25c87180-c127-11ea-99da-972ac47a9894.png)

![image](https://user-images.githubusercontent.com/37310063/86950960-4264a980-c127-11ea-8162-a5771ff9bedb.png)

**Comment by betefaber** [comment]
The error remains in v0.5.0-alpha.4

**Comment by betefaber** [comment]
Tested in **v0.4.4-rc.1** (k8s)

- add remote node
```
curl -X POST ${DOJOT_HOST}/flows/v1/node -H "Authorization: Bearer ${JWT}" -H "content-type: application/json" -d '{"image": "dojot/kelvin-example:3.0.0-alpha2","id":"kelvin"}'
{"message":"ok"}
```
```
<19:06:17 23/10/2020> -- |nodeMngr -- DEBUG: Loading remote nodes for tenant admin ...
<19:06:17 23/10/2020> -- |nodeMngr -- DEBUG: ...Loaded remote nodes for tenant admin.
<19:06:17 23/10/2020> -- |nodeMngr -- DEBUG: Succeeded to set manager to handle processing nodes for tenant admin
<19:06:18 23/10/2020> -- |amqp -- INFO: Producer ready ...
<19:06:18 23/10/2020> -- |amqp -- INFO: consumer { consumerTag: 'amq.ctag-Y4GkoUaBu_FkbmYiPc5eWg' } ready ...
<19:06:18 23/10/2020> -- |amqp -- INFO: Producer ready ...
<19:06:18 23/10/2020> -- |amqp -- INFO: Producer ready ...
<19:06:18 23/10/2020> -- |amqp -- INFO: Producer ready ...
<19:06:18 23/10/2020> -- |amqp -- INFO: Producer ready ...
<19:06:18 23/10/2020> -- |amqp -- INFO: Producer ready ...
<19:06:18 23/10/2020> -- |amqp -- INFO: Producer ready ...
<19:06:18 23/10/2020> -- |amqp -- INFO: Producer ready ...
<19:06:18 23/10/2020> -- |amqp -- INFO: Producer ready ...
<19:06:18 23/10/2020> -- DEBUG: Connections established with RabbitMQ!
<19:06:18 23/10/2020> -- |amqp -- INFO: Producer ready ...
<19:06:18 23/10/2020> -- |amqp -- INFO: Producer ready ...
<19:06:18 23/10/2020> -- |amqp -- INFO: Producer ready ...
<19:06:18 23/10/2020> -- |amqp -- INFO: Producer ready ...
<19:06:18 23/10/2020> -- |amqp -- INFO: Producer ready ...
<19:06:18 23/10/2020> -- |amqp -- INFO: consumer { consumerTag: 'amq.ctag-moQXcydGfCa9h4WlVOY9_Q' } ready ...
<19:06:18 23/10/2020> -- |amqp -- INFO: consumer { consumerTag: 'amq.ctag-ImAIP0pFvGyXFKVx-M5_Pg' } ready ...
<19:06:18 23/10/2020> -- |amqp -- INFO: consumer { consumerTag: 'amq.ctag-LJyGhhjsal9UwFnap_P6bg' } ready ...
<19:06:18 23/10/2020> -- |amqp -- INFO: Producer ready ...
<19:06:18 23/10/2020> -- |amqp -- INFO: Producer ready ...
<19:06:18 23/10/2020> -- |amqp -- INFO: Producer ready ...
<19:06:18 23/10/2020> -- |amqp -- INFO: Producer ready ...
<19:06:18 23/10/2020> -- |amqp -- INFO: consumer { consumerTag: 'amq.ctag-J4mlshfJC_kAe9BGav0jkQ' } ready ...
<19:06:18 23/10/2020> -- |amqp -- INFO: consumer { consumerTag: 'amq.ctag-D0KRxgymelciJrvjFRxxZQ' } ready ...
<19:06:18 23/10/2020> -- |amqp -- INFO: Producer ready ...
<19:06:18 23/10/2020> -- |amqp -- INFO: Producer ready ...
<19:06:18 23/10/2020> -- |amqp -- INFO: consumer { consumerTag: 'amq.ctag-_UpuRGn62CwsmoxSjqIPJw' } ready ...
<19:06:18 23/10/2020> -- |amqp -- INFO: consumer { consumerTag: 'amq.ctag-UXZ8pXGKU53OnHhzZnNp1A' } ready ...
<19:06:18 23/10/2020> -- |amqp -- INFO: consumer { consumerTag: 'amq.ctag-DhJA3q9R6nWuVg58iPTfCA' } ready ...
<19:06:18 23/10/2020> -- |amqp -- INFO: consumer { consumerTag: 'amq.ctag-vS9Rp1Q4oz56OHh1Xu4zPw' } ready ...
<19:12:23 23/10/2020> -- |kb8sRemoveNode -- DEBUG: Using kubernetes driver.
<19:12:23 23/10/2020> -- |kb8sRemoveNode -- DEBUG: Selected engine: kubernetes
<19:12:23 23/10/2020> -- |kb8sRemoveNode -- DEBUG: Using kubernetes API servera @ https://10.96.0.1:443
<19:12:23 23/10/2020> -- |kb8sRemoveNode -- DEBUG: Testing access...
<19:12:23 23/10/2020> -- |kb8sRemoveNode -- DEBUG: Retrieving current deployment...
<19:12:23 23/10/2020> -- |kb8sRemoveNode -- DEBUG: Sending request to server...
<19:12:23 23/10/2020> -- |kb8sRemoveNode -- DEBUG: ... request was sent to the server.
<19:12:23 23/10/2020> -- |kb8sRemoveNode -- DEBUG: Retrieving current deployment...
<19:12:23 23/10/2020> -- |kb8sRemoveNode -- DEBUG: Sending request to server...
<19:12:23 23/10/2020> -- |kb8sRemoveNode -- DEBUG: ... request was sent to the server.
<19:12:24 23/10/2020> -- |kb8sRemoveNode -- DEBUG: Current flowbroker deployments are: { statusCode: 200,
 ...
<19:12:24 23/10/2020> -- |kb8sRemoveNode -- DEBUG: ... server access is OK.
<19:12:24 23/10/2020> -- |kb8sRemoveNode -- DEBUG: Current flowbroker deployments are: { statusCode: 200,
...
<19:12:24 23/10/2020> -- |kb8sRemoveNode -- DEBUG: Building deployment creation request...
<19:12:24 23/10/2020> -- |kb8sRemoveNode -- DEBUG: Adding container dojot/kelvin-example:3.0.0-alpha2 to the set...
<19:12:24 23/10/2020> -- |kb8sRemoveNode -- DEBUG: ... container adminkelvin was added to the set.
<19:12:24 23/10/2020> -- |kb8sRemoveNode -- DEBUG: ... deployment creation request was built.
<19:12:24 23/10/2020> -- |kb8sRemoveNode -- DEBUG: Deployment is:
<19:12:24 23/10/2020> -- |kb8sRemoveNode -- DEBUG: { apiVersion: 'apps/v1',
  kind: 'Deployment',
  metadata:
   { labels: { app: 'flowbroker', name: 'flownode-adminkelvin' },
     name: 'flownode-adminkelvin' },
  spec: (96%)
   { selector:
      { matchLabels: { app: 'flowbroker', name: 'flownode-adminkelvin' } },
     replicas: 1,
     template:
      { metadata: { labels: { name: 'flownode-adminkelvin', app: 'flowbroker' } },
        spec:
         { containers:
            [ { name: 'adminkelvin',
                image: 'dojot/kelvin-example:3.0.0-alpha2',
                imagePullPolicy: 'Always',
                ports: [ { name: 'amqp', port: 5555, containerPort: 5555 } ] } ],
           restartPolicy: 'Always' } } } }
<19:12:24 23/10/2020> -- |kb8sRemoveNode -- DEBUG: Sending request to server...
<19:12:24 23/10/2020> -- |kb8sRemoveNode -- DEBUG: ... request was sent to the server.
<19:12:24 23/10/2020> -- |kb8sRemoveNode -- DEBUG: Creating service for this deployment...
<19:12:24 23/10/2020> -- |kb8sRemoveNode -- DEBUG: Service to be created: { apiVersion: 'v1',
  kind: 'Service',
  metadata:
   { name: 'flownode-adminkelvin',
     labels: { app: 'flowbroker', name: '' } },
  spec:
   { selector: { name: 'flownode-adminkelvin' },
     ports: [ { protocol: 'TCP', port: 5555, targetPort: 5555 } ] } }
<19:12:24 23/10/2020> -- |kb8sRemoveNode -- DEBUG: ... service for deployment created:  { statusCode: 201,
  body:
   { kind: 'Service',
     apiVersion: 'v1',
     metadata:
      { name: 'flownode-adminkelvin',
        namespace: 'dojot',
        selfLink: '/api/v1/namespaces/dojot/services/flownode-adminkelvin',
        uid: '7632f8d1-ed97-4226-b698-397abd2ddbb0',
        resourceVersion: '18207518',
        creationTimestamp: '2020-10-23T19:12:24Z',
        labels: { app: 'flowbroker', name: '' } },
     spec:
      { ports: [ { protocol: 'TCP', port: 5555, targetPort: 5555 } ],
        selector: { name: 'flownode-adminkelvin' },
        clusterIP: '10.96.210.101',
        type: 'ClusterIP',
        sessionAffinity: 'None' },
     status: { loadBalancer: {} } } }
<19:12:24 23/10/2020> -- |dispatcher -- INFO: Dispatcher connected to flownode-adminkelvin on port 5555
<19:12:29 23/10/2020> -- |dispatcher -- DEBUG: Received reply: {"payload":{"payload":{"id":"sample/kelvin","name":"kelvin","module":"kelvin","version
":"0.0.0"}},"requestId":"4d26ea73-95e7-47e3-9ea5-34d28ce07c08"}
<19:12:29 23/10/2020> -- |dispatcher -- DEBUG: Received reply: {"payload":{"payload":"<script type=\"text/x-red\" data-template-name=\"kelvin\">\n
 <div class=\"form-row\">\n        <label for=\"node-input-name\"><i class=\"fa fa-tag\"></i> Name</label>\n        <input type=\"text\" id=\"node-input-name\">\
n    </div>\n    <div class=\"form-row\">\n        <label for=\"node-input-in\"><i class=\"fa fa-edit\"></i> <span>input from</span></label>\n        <input type
=\"text\" id=\"node-input-in\" placeholder=\"payload\" style=\"width:250px;\">\n        <input type=\"hidden\" id=\"node-input-inFieldType\">\n    </div>\n    <d
iv class=\"form-row\">\n        <label for=\"node-input-out\"><i class=\"fa fa-edit\"></i> <span>output to</span></label>\n        <input type=\"text\" id=\"node
-input-out\" placeholder=\"payload\" style=\"width:250px;\">\n        <input type=\"hidden\" id=\"node-input-outFieldType\">\n    </div>\n</script>\n\n<script ty
pe=\"text/x-red\" data-help-name=\"kelvin\">\n</script>\n\n<script type=\"text/javascript\">\n    RED.nodes.registerType('kelvin', {\n        category: 'converte
rs',\n        color: \"rgb(231, 231, 174)\",\n        defaults: {\n            name: { value: \"\", required: false },\n            out:  { value: \"payload\", v
alidate: RED.validators.typedInput(\"outFieldType\") },\n            outFieldType: { value: \"msg\" },\n            in: { value: \"payload\", validate: RED.valid
ators.typedInput(\"inFieldType\") },\n            inFieldType: { value: \"msg\" },\n        },\n        inputs: 1,\n        outputs: 1,\n        icon: \"cog.png\
",\n        label: function () {\n            return this.name || \"to kelvin\";\n        },\n        labelStyle: function () {\n            return this.name ? \
"node_label_italic\" : \"\";\n        },\n        oneditprepare: function () {\n            if (!this.outFieldType) {\n                this.outFieldType = 'msg';
\n            }\n\n            $(\"#node-input-out\").typedInput({\n                default: 'msg',\n                types: ['msg'],\n                typeField:
$(\"#node-input-outFieldType\")\n            });\n\n            if (!this.inFieldType) {\n                this.inFieldType = 'msg';\n            }\n\n
 $(\"#node-input-in\").typedInput({\n                default: 'msg',\n                types: ['msg'],\n                typeField: $(\"#node-input-inFieldType\")\
n            });\n        },\n        oneditsave: function () { }\n    });\n</script>"},"requestId":"19352af6-250c-44ce-82fe-0a5ffb138a03"}
<19:12:29 23/10/2020> -- |nodeMngr -- DEBUG: Succeeded to add remote node with id kelvin
```
- list remote node
```
curl -X GET "${DOJOT_HOST}/flows/v1/node" -H "Authorization: Bearer ${JWT}" -H "Content-Type:application/json"
{"nodes":[{"id":"kelvin","image":"dojot/kelvin-example:3.0.0-alpha2"}]}
```
![image](https://user-images.githubusercontent.com/37310063/97050212-5952ce80-1553-11eb-8a0f-0f1ed84aa5ca.png)

**Comment by betefaber** [comment]
Needs to be tested on v0.5.0-rc.1.

**Comment by betefaber** [comment]
Fixed in **v0.5.0-rc.1**

- adding a emote node
```
curl -X POST ${DOJOT_HOST}/flows/v1/node -H "Authorization: Bearer ${JWT}" -H "content-type: application/json" -d '{"image": "dojot/kelvin-example:3.0.0-alpha2","id":"kelvin"}'
{"message":"ok"}
```
```
<20:26:38 11/11/2020> -- |kb8sRemoveNode -- DEBUG: Building deployment creation request...
<20:26:38 11/11/2020> -- |kb8sRemoveNode -- DEBUG: Adding container dojot/kelvin-example:3.0.0-alpha2 to the set...
<20:26:38 11/11/2020> -- |kb8sRemoveNode -- DEBUG: ... container adminkelvin was added to the set.
<20:26:38 11/11/2020> -- |kb8sRemoveNode -- DEBUG: ... deployment creation request was built.
<20:26:38 11/11/2020> -- |kb8sRemoveNode -- DEBUG: Deployment is:
<20:26:38 11/11/2020> -- |kb8sRemoveNode -- DEBUG: { apiVersion: 'apps/v1',
  kind: 'Deployment',
  metadata:
   { labels: { app: 'flowbroker', name: 'flownode-adminkelvin' },
     name: 'flownode-adminkelvin' },
  spec:
   { selector:
      { matchLabels: { app: 'flowbroker', name: 'flownode-adminkelvin' } },
     replicas: 1,
     template:
      { metadata: { labels: { name: 'flownode-adminkelvin', app: 'flowbroker' } },
        spec:
         { containers:
            [ { name: 'adminkelvin',
                image: 'dojot/kelvin-example:3.0.0-alpha2',
                imagePullPolicy: 'Always',
                ports: [ { name: 'amqp', port: 5555, containerPort: 5555 } ] } ],
           restartPolicy: 'Always' } } } }
<20:26:38 11/11/2020> -- |kb8sRemoveNode -- DEBUG: Sending request to server...
<20:26:38 11/11/2020> -- |kb8sRemoveNode -- DEBUG: ... request was sent to the server.
<20:26:38 11/11/2020> -- |kb8sRemoveNode -- DEBUG: ... server access is OK.
<20:26:38 11/11/2020> -- |kb8sRemoveNode -- DEBUG: Creating service for this deployment...
<20:26:38 11/11/2020> -- |kb8sRemoveNode -- DEBUG: Service to be created: { apiVersion: 'v1',
  kind: 'Service',
  metadata:
   { name: 'flownode-adminkelvin',
     labels: { app: 'flowbroker', name: '' } },
  spec:
   { selector: { name: 'flownode-adminkelvin' },
     ports: [ { protocol: 'TCP', port: 5555, targetPort: 5555 } ] } }
<20:26:38 11/11/2020> -- |kb8sRemoveNode -- DEBUG: ... service for deployment created:  { statusCode: 201,
  body:
   { kind: 'Service',
     apiVersion: 'v1',
     metadata:
      { name: 'flownode-adminkelvin',
        namespace: 'dojot',
        selfLink: '/api/v1/namespaces/dojot/services/flownode-adminkelvin',
        uid: 'fd3e9488-d0c4-449d-ae28-80e3d11a68c7',
        resourceVersion: '22031338',
        creationTimestamp: '2020-11-11T20:26:38Z',
        labels: { app: 'flowbroker', name: '' } },
     spec:
      { ports: [ { protocol: 'TCP', port: 5555, targetPort: 5555 } ],
        selector: { name: 'flownode-adminkelvin' },
        clusterIP: '10.96.169.239',
        type: 'ClusterIP',
        sessionAffinity: 'None' },
     status: { loadBalancer: {} } } }
<20:26:38 11/11/2020> -- |dispatcher -- INFO: Dispatcher connected to flownode-adminkelvin on port 5555
<20:26:44 11/11/2020> -- |dispatcher -- DEBUG: Received reply: {"payload":{"payload":{"id":"sample/kelvin","name":"kelvin","module":"kelvin","version":"0.0.0"}},"requestId":"3899aa7a-23f4-44f2-831e-3172d08ae185"}
<20:26:44 11/11/2020> -- |dispatcher -- DEBUG: Received reply: {"payload":{"payload":"<script type=\"text/x-red\" data-template-name=\"kelvin\">\n    <div class=\"form-row\">\n        <label for=\"node-input-name\"><i class=\"fa fa-tag\"></i> Name</label>\n        <input type=\"text\" id=\"node-input-name\">\n    </div>\n    <div class=\"form-row\">\n        <label for=\"node-input-in\"><i class=\"fa fa-edit\"></i> <span>input from</span></label>\n        <input type=\"text\" id=\"node-input-in\" placeholder=\"payload\" style=\"width:250px;\">\n        <input type=\"hidden\" id=\"node-input-inFieldType\">\n    </div>\n    <div class=\"form-row\">\n        <label for=\"node-input-out\"><i class=\"fa fa-edit\"></i> <span>output to</span></label>\n        <input type=\"text\" id=\"node-input-out\" placeholder=\"payload\" style=\"width:250px;\">\n        <input type=\"hidden\" id=\"node-input-outFieldType\">\n    </div>\n</script>\n\n<script type=\"text/x-red\" data-help-name=\"kelvin\">\n</script>\n\n<script type=\"text/javascript\">\n    RED.nodes.registerType('kelvin', {\n        category: 'converters',\n        color: \"rgb(231, 231, 174)\",\n        defaults: {\n            name: { value: \"\", required: false },\n            out:  { value: \"payload\", validate: RED.validators.typedInput(\"outFieldType\") },\n            outFieldType: { value: \"msg\" },\n            in: { value: \"payload\", validate: RED.validators.typedInput(\"inFieldType\") },\n            inFieldType: { value: \"msg\" },\n        },\n        inputs: 1,\n        outputs: 1,\n        icon: \"cog.png\",\n        label: function () {\n            return this.name || \"to kelvin\";\n        },\n        labelStyle: function () {\n            return this.name ? \"node_label_italic\" : \"\";\n        },\n        oneditprepare: function () {\n            if (!this.outFieldType) {\n                this.outFieldType = 'msg';\n            }\n\n            $(\"#node-input-out\").typedInput({\n                default: 'msg',\n                types: ['msg'],\n                typeField: $(\"#node-input-outFieldType\")\n            });\n\n            if (!this.inFieldType) {\n                this.inFieldType = 'msg';\n            }\n\n            $(\"#node-input-in\").typedInput({\n                default: 'msg',\n                types: ['msg'],\n                typeField: $(\"#node-input-inFieldType\")\n            });\n        },\n        oneditsave: function () { }\n    });\n</script>"},"requestId":"8a6b0b64-fb58-466d-87f3-761947d693b8"}
<20:26:44 11/11/2020> -- |nodeMngr -- DEBUG: Succeeded to add remote node with id kelvin
```

- removing a remote node
```
curl -X DELETE ${DOJOT_HOST}/flows/v1/node/kelvin -H "Authorization: Bearer ${JWT}" -H "content-type: application/json"
{"message":"ok"}
```

```
<20:35:19 11/11/2020> -- |nodeMngr -- DEBUG: Trying to remove docker container undefined
<20:35:19 11/11/2020> -- |kb8sRemoveNode -- DEBUG: Retrieving current deployment...
<20:35:19 11/11/2020> -- |kb8sRemoveNode -- DEBUG: Sending request to server...
<20:35:19 11/11/2020> -- |kb8sRemoveNode -- DEBUG: ... request was sent to the server.
<20:35:19 11/11/2020> -- |kb8sRemoveNode -- DEBUG: Removing deployment adminkelvin...
<20:35:19 11/11/2020> -- |kb8sRemoveNode -- DEBUG: Removing container adminkelvin from the set.
<20:35:19 11/11/2020> -- |kb8sRemoveNode -- DEBUG: Current container list is
<20:35:19 11/11/2020> -- |kb8sRemoveNode -- DEBUG: Scaling down deployment flownode-adminkelvin...
<20:35:19 11/11/2020> -- |kb8sRemoveNode -- DEBUG: ... deployment scale down request was sent to the server.
<20:35:19 11/11/2020> -- |kb8sRemoveNode -- DEBUG: ... deployment flownode-adminkelvin was scaled down.
<20:35:19 11/11/2020> -- |kb8sRemoveNode -- DEBUG: Removing deployment flownode-adminkelvin...
<20:35:19 11/11/2020> -- |kb8sRemoveNode -- DEBUG: ... deployment removal request was sent to the server.
<20:35:19 11/11/2020> -- |kb8sRemoveNode -- DEBUG: ... deployment flownode-adminkelvin was removed.
<20:35:19 11/11/2020> -- |kb8sRemoveNode -- DEBUG: Removing service for this deployment...
<20:35:19 11/11/2020> -- |kb8sRemoveNode -- DEBUG: ... Service for deployment removed:  { statusCode: 200,
  body:
   { kind: 'Status',
     apiVersion: 'v1',
     metadata: {},
     status: 'Success',
     details:
      { name: 'flownode-adminkelvin',
        kind: 'services',
        uid: 'fd3e9488-d0c4-449d-ae28-80e3d11a68c7' } } }
<20:35:19 11/11/2020> -- |nodeMngr -- DEBUG: Succeeded to remove container.
<20:35:19 11/11/2020> -- |nodeMngr -- DEBUG: Succeeded to delete remote node with id kelvin for tenant admin.

```
